components/NavLink.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import cx from 'clsx';
+
+function NavLink({
+  href,
+  className,
+  activeClassName,
+  external,
+  target,
+  rel,
+  children,
+}) {
+  const router = useRouter();
+  return external ? (
+    <a
+      href={href}
+      target={target || '_blank'}
+      rel={rel || 'noopener noreferrer'}
```

Comment:
I think target-blank and no-referrer by default is a bit opinionated. Since `NavLink` is mostly used for navigation within Cofacts, we should not set default for these values.

Although we are removing the defaults so it's not used here, I would like to mention that it is preferred to set default props (if any) using object destructuring default value `function NavLink({prop1 = 'defaultValue1'})`. It has the following benefit:
1. Default value declared with prop, more declarative
2. Editors that pick up function definition (like VS Code) can show default when invoking the function / component
3. [Storybook addon info](https://github.com/storybookjs/storybook/tree/master/addons/info) can automatically pick up default values in its props table

Reply:
Sorry I missed the `external` part. Please ignore the comment above